src/views/Player.vue

```diff
@@ -279,6 +325,33 @@ export default {
           .then(response => response.json())
           .then(data => console.log(data))
           .catch(err => console.log(err))
+        
+      
+      if (this.isTutorialUploadRequired) {
+          const tutorial_progress = {
+              'user-id': this.userId,
+              'configs': {
+                'tutorial': {
+                    'isComplete': this.isTutorialComplete,
+                    'progress': this.tutorialProgress
+                }
+              }
+          }
+
+          const json_tutorial_progress = JSON.stringify(tutorial_progress)
+
+          fetch(process.env.VUE_APP_BACKEND + 
+                process.env.VUE_APP_BACKEND_UPDATE_USER_CONFIG, {method: 'POST', body: json_tutorial_progress,
+                headers: {
+                  'Accept': 'application/json',
+                  'Content-Type': 'application/json'
+                }})
+                .then(response => response.json())
+                .then(data => console.log(data))
+                .catch(err => console.log(err))
+          
+          if (this.isTutorialComplete) this.isTutorialUploadRequired = false;
+      }
```

Comment:
Uploading the updated config (with updated tutorial data)

Reply:
I feel like this condition `isTutorialUploadRequired` should not be needed. If the tutorial is complete, none of this code should be reached because some `if` higher in the chain will block tutorial related actions. But for future